pettingzoo/classic/tictactoe/test_board.py

```diff
@@ -113,16 +112,16 @@ def test_tictactoe_bad_move() -> None:
     board = Board()
     # 1) move out of bounds should be rejected
     for outside_space in [-1, 9]:
-        with pytest.raises(BadTicTacToeMoveException):
+        with pytest.raises(AssertionError):
             board.play_turn(0, outside_space)
 
     # 2) move by unknown agent should be rejected
     for unknown_agent in [-1, 2]:
-        with pytest.raises(BadTicTacToeMoveException):
+        with pytest.raises(AssertionError):
```

Comment:
Why pytest raises? Haven’t seen that before

Reply:
As far as I know, that's the [recommended pytest way](https://docs.pytest.org/en/8.0.x/how-to/assert.html#assertions-about-expected-exceptions) to confirm that a block of code raises a specific assertion. It catches and ignores that exception (because it's expected) but raises an error if the code in that block doesn't raise the expected error.

Reply:
Ok fair, we should probably also go and make the code adhere to that as well, because I’m pretty sure elsewhere that isn’t done, but I can see it being good practice. Not a problem here tbh since it’s so small I think it’s fine to have here and then we can do an issue or future prs to fix other stuff to adhere to this practice